qiime2/core/testing/method.py

```diff
@@ -138,3 +138,11 @@ def type_match_list_and_set(ints: list, strs1: list, strs2: set) -> list:
 
 def union_inputs(ints1: Union[dict, list], ints2: list) -> list:
     return [0]
+
+
+def list_of_ints(ints: list) -> list:
```

Comment:
Forgot about how IntSequence works. I think the type here for a collection of them would actually be:
`typing.List[list]` (you can access the inner fields by using `.__args__` -> `(list,)`)


Alternatively, we may want to disallow list and dict in transformers, although it will be a huge pain to rewrite these test actions, so if it can be done without that, I'm all for it.